src/dynamic_graph_head/controllers.py

```diff
@@ -75,10 +77,23 @@ def map_sliders(self, sliders):
                 if i >= 2:
                     sliders_out[2 * i + 0] *= -1
                     sliders_out[2 * i + 1] *= -1
+        
+        elif self.joint_positions.shape[0] == 6:  # bolt
+            slider_A = sliders[0]
+            slider_B = sliders[1]
+            slider_C = sliders[2]
+
+            for i in range(2):
+                sliders_out[3 * i + 0] = slider_A 
+                sliders_out[3 * i + 1] = slider_B 
+                sliders_out[3 * i + 2] = slider_C 
```

Comment:
This works. If possible I would use only two slider for this motion (we use the two-sliders in New York).

Can you change this to a similar line as in `Solo12`:

```
sliders_out[3 * i + 2] = 2. * (1. - slider_B)
```

?